.github/workflows/pylint.yml

```diff
@@ -27,4 +27,4 @@ jobs:
     - name: Analysing the code with pylint
       run: |
         # TODO #682
-        pylint --disable=fixme,invalid-name,missing-function-docstring,missing-class-docstring,too-many-arguments,too-many-locals,too-many-instance-attributes,too-few-public-methods,protected-access,too-many-statements,duplicate-code,too-many-branches $(git ls-files '*.py')
+        pylint --disable=fixme,invalid-name,missing-function-docstring,missing-class-docstring,too-many-arguments,too-many-locals,too-many-instance-attributes,too-few-public-methods,protected-access,too-many-statements,duplicate-code,too-many-branches,C0330,C0326 --max-line-length=88 $(git ls-files '*.py')
```

Comment:
This change is why the Pylint checks now fail as max line length is reduced to 88 (from 100).

Reply:
do we need to modify it manually?

Reply:
no - if we run the precommit hooks the lines will automatically be reformatted to adhere to <= 88 character length. However we will have I think 200+ file changes to commit.

Reply:
I suggest then, we wait with merging for the breakup PR merge